backends/bmv2/common/JsonObjects.cpp

```diff
@@ -385,12 +385,14 @@ JsonObjects::add_parser_transition_key(const unsigned state_id, Util::IJson* new
 }
 
 void
-JsonObjects::add_parse_vset(const cstring& name, const unsigned size) {
+JsonObjects::add_parse_vset(const cstring& name, const unsigned& size,
+                            const big_int& n) {
```

Comment:
I suggest renaming the parameters from `size` to `bitwidth` and from `n` to `size`.